share/completions/minikube.fish

```diff
@@ -0,0 +1,79 @@
+    if test (count $cmd) -gt 1
+        if test $argv[1] = $cmd[2]
+            return 0
+        end
+    end
+    return 1
+end
+
+complete -c minikube -f -n "__minikube_no_command" -a addons -d "Modify minikube's kubernetes addons"
+complete -c minikube -f -n "__minikube_no_command" -a completion -d "Outputs minikube shell completion for the given shell (bash)"
+complete -c minikube -f -n "__minikube_no_command" -a config -d "Modify minikube config"
+complete -c minikube -f -n "__minikube_no_command" -a dashboard -d "Opens/displays the kubernetes dashboard URL for your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a delete -d "Deletes a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a docker-env -d "sets up docker env variables; similar to '\$(docker-machine env)'"
+complete -c minikube -f -n "__minikube_no_command" -a get-k8s-versions -d "Gets the list of available kubernetes versions available for minikube."
+complete -c minikube -f -n "__minikube_no_command" -a ip -d "Retrieve the IP address of the running cluster."
+complete -c minikube -f -n "__minikube_no_command" -a logs -d "Gets the logs of the running localkube instance, used for debugging minikube, not user code."
+complete -c minikube -f -n "__minikube_no_command" -a service -d "Gets the kubernetes URL(s) for the specified service in your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a ssh -d "Log into or run a command on a machine with SSH; similar to 'docker-machine ssh'"
+complete -c minikube -f -n "__minikube_no_command" -a start -d "Starts a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a status -d "Gets the status of a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a stop -d "Stops a running local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a version -d "Print the version of minikube."
+complete -c minikube -l "alsologtostderr" -d "log to standard error as well as files"
+complete -c minikube -l "log_backtrace_at" -d "when logging hits line file:N, emit a stack trace (default :0)"
+complete -c minikube -l "log_dir" -d "If non-empty, write log files in this directory (default \"\")"
+complete -c minikube -l "logtostderr" -d "log to standard error instead of files"
+complete -c minikube -l "show-libmachine-logs" -d "Deprecated: To enable libmachine logs, set --v=3 or higher"
+complete -c minikube -l "stderrthreshold" -d "logs at or above this threshold go to stderr (default 2)"
+complete -c minikube -s "v" -d "log level for V logs"
+complete -c minikube -l "v" -d "log level for V logs"
+complete -c minikube -l "vmodule" -d "comma-separated list of pattern=N settings for file-filtered logging"
+complete -c minikube -n "__minikube_using_command addons" -l "format" -d "Go template format string for the addon list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/"
```

Comment:
Perhaps we should even skip the part about the format and URL. I guess completions might not be the place to get that information. What do you think?